docs/platforms/react-native/troubleshooting/apple-privacy-manifest.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Apple Privacy Manifest
+description: "Troubleshoot and resolve common issues with the Apple Privacy Manifest and Sentry React Native SDK."
+sidebar_order: 50
+---
+
+Sentry React Native SDKs provides errors reporting and performanceEntry monitoring for React Native applications. The SDKs automatically collect errors and crashes in your application and send them to Sentry. To do this, the SDKs need to access certain information about the device and the application. Some of the used APIs are considered sensitive by Apple and require the SDK to provide a privacy manifest. For more details read the [Data Privacy for Mobile](/product/security/mobile-privacy/).
+
+If you are using dynamically linked frameworks, Apple will automatically process the SDK's privacy manifest. If you are using statically linked libraries, which is the default for React Native applications, you need to provide the privacy manifest yourself. This guide will help you create the privacy manifest for your React Native and Expo applications.
+
+## Create Privacy Manifest in Xcode
+
+First you need to create a privacy manifest file in your Xcode project. The file should be named `PrivacyInfo.xcprivacy` and should be placed in the root of your Xcode project. Follow the [Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files#4284009) guide by Apple to create the file.
```

Comment:
`h`: we need to point out that the privacy manifest provided this way needs to cover all data usages (via `NSPrivacyCollectedDataTypes`) and api usages (via `NSPrivacyAccessedAPITypes`) for **the whole app** including all libraries. the templates below cover the current status of api usages for our SDK alone